gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,11 +1,5 @@
-#
-# Copyright OpenSearch Contributors
-# SPDX-License-Identifier: Apache-2.0
-#
```

Comment:
Isn't the copyright required for .properties files?

Reply:
I don't believe we should have it here because this is a file which was generated by Gradle itself.

Reply:
Though, it does appear that the OpenSearch core project includes it.

https://github.com/opensearch-project/OpenSearch/blob/c41b41140b745fa62b38afe7ead27002d77e809f/gradle/wrapper/gradle-wrapper.properties#L1-L10

Reply:
@asifsmohammed , I added the header back to match what OpenSearch core does.